frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
```

Comment:
IMO this one is obsolete - `eslint-plugin-prettier` already does the job and does it better since it has an actual plugin, the other one just has a set of rules that seem to overlap for 99%.